src/components/intent.ts

```diff
@@ -707,7 +707,7 @@ export class Intent {
      */
     public async uploadContent(content: Buffer|string|ReadStream, opts: FileUploadOpts = {}): Promise<string> {
         await this.ensureRegistered();
-        return this.client.uploadContent(content, {...opts, onlyContentUri: true});
+        return this.client.uploadContent(content, {...opts, rawResponse: false, onlyContentUri: true});
```

Comment:
Looks good :+1: Matches API docs https://matrix-org.github.io/matrix-js-sdk/9.1.0/module-base-apis.MatrixBaseApis.html#uploadContent

And ran into this recently in https://gitlab.com/gitterHQ/webapp/-/merge_requests/2042/diffs#note_445605432